dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -69,12 +69,14 @@ def find_airflow_sources_root():
     os.chdir(__AIRFLOW_SOURCES_ROOT)
 
 
+find_airflow_sources_root()
```

Comment:
BTW. This something we will have to add later - but I will have to think how to do it best. this is just to get the example tests working.

Reply:
Could you explain more about this part @potiuk ? I couldn't understand it.